src/utils/aotf.js

```diff
@@ -161,8 +161,8 @@ export const cylcObjects = Object.freeze({
   User: 'user',
   Workflow: 'workflow',
   CyclePoint: 'cycle',
-  Namespace: 'task',
-  // Task: 'task',
+  Family: 'family',
```

Comment:
Namespace is correct, it is ambiguous for task or family, e.g. see `cylc broadcast -n` & `cylc graph -n`.

Reply:
We must have a way of distinguishing family commands from task commands.